### ISSUE TITLE
Crash on startup on dedicated server

### ISSUE THREAD

**Issue by ExpensiveKoala** [Daomephsta/LootTweaker#53]
**Description:**
https://github.com/Daomephsta/LootTweaker/blob/1.12/src/main/java/leviathan143/loottweaker/common/tweakers/LootTableTweaker.java#L76

On a dedicated server, I18n doesn't exist so during class loading, it will produce an error. It's weird, because it worked fine before but only now is it saying it's an issue.

**Version Info (Exact versions only):**  
LootTweaker:  0.0.9
CraftTweaker/Minetweaker:  4.1.18
Forge:  14.23.5.2838
Minecraft:  1.12.2

**Logs, scripts and other useful information:**  
```
java.lang.NoClassDefFoundError: net/minecraft/client/resources/I18n
        at leviathan143.loottweaker.common.tweakers.LootTableTweaker.onTableLoadingStarted(LootTableTweaker.java:76) ~[LootTableTweaker.class:?]
        at leviathan143.loottweaker.common.tweakers.LootTableTweaker.onTableLoad(LootTableTweaker.java:67) ~[LootTableTweaker.class:?]
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_127_LootTableTweaker_onTableLoad_LootTableLoadEvent.invoke(.dynamic) ~[?:?]
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
```

**Comment by Daomephsta** [comment]
Oops! You can downgrade to 0.0.8 until this is fixed, it shouldn't have this issue.

**Comment by Daomephsta** [comment]
The fix has been released in 0.0.10. Currently waiting on CurseForge approval.
